modular_skyrat/modules/modular_reagents/code/reagents/medicine.dm

```diff
@@ -25,6 +25,84 @@
 	M.adjustOrganLoss(ORGAN_SLOT_HEART,3 * REM * delta_time, 80)
 	..()
 
+
+/datum/reagent/medicine/ataraxydone
+	name = "Ataraxydone"
+	description = "A stasis-inducing chemical used for exotic surgeries or emergency medicine."
+	taste_description = "liquid"
+	reagent_state = LIQUID
+	color = "#e0d394" // 224, 211, 148
+	metabolization_rate = 0.2 * REAGENTS_METABOLISM
+	overdose_threshold = 20
+	ph = 7
+	purity = REAGENT_STANDARD_PURITY
+	creation_purity = REAGENT_STANDARD_PURITY
+	impure_chem = /datum/reagent/impurity/ataraxydone
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED | REAGENT_DEAD_PROCESS | REAGENT_IGNORE_STASIS
+
+/datum/reagent/medicine/ataraxydone/on_mob_dead(mob/living/carbon/owner, delta_time)
+	.=..()
+	owner.apply_status_effect(STATUS_EFFECT_STASIS_MOBILE, STASIS_CHEMICAL_EFFECT)
+	metabolization_rate = 0.1 * REAGENTS_METABOLISM
+
+/datum/reagent/medicine/ataraxydone/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	var/ataraxydone_speed = max(1, min(current_cycle/10, 8))
+	if(owner.resting || owner.stat >= SOFT_CRIT)
+		owner.apply_status_effect(STATUS_EFFECT_STASIS_MOBILE, STASIS_CHEMICAL_EFFECT)
+		owner.add_movespeed_modifier(/datum/movespeed_modifier/reagent/ataraxyprone)
+		current_cycle += 1
+	else
+		owner.remove_status_effect(STATUS_EFFECT_STASIS_MOBILE, STASIS_CHEMICAL_EFFECT)
+		owner.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/ataraxyprone)
+	if(current_cycle >= 10)
```

Comment:
This should be a define too. And those kinds of defines should be undefined at the end of the file, using `#undef DEFINE_NAME`